JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+  agent {
+    node {
+      label 'project:any'
+    }
+  }
+  stages {
+    stage('Clean Workspace') {
+      steps {
+        cleanWs()
+      }
+    }
+    stage('Git Clone') {
+      steps {
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']],
+        doGenerateSubmoduleConfigurations: false,
+        extensions: [],
+        submoduleCfg: [],
+        userRemoteConfigs: [[credentialsId: 'CIDA-Jenkins-GitHub',
+        url: 'https://github.com/usgs/aqts-capture-db.git']]])
+      }
+    }
+    stage('Download liquibase jar') {
+      steps {
+        sh '''mkdir $WORKSPACE/rc
+          if [ ! -f rc/liquibase ]; then
+            /usr/local/bin/aws s3 cp s3://owi-common-resources/resources/InstallFiles/liquibase/liquibase-$LIQUIBASE_VERSION.tar.gz $WORKSPACE/rc/liquibase.tar.gz
+            /usr/bin/tar xzf $WORKSPACE/rc/liquibase.tar.gz --overwrite -C $WORKSPACE/rc
+            /usr/local/bin/aws s3 cp s3://owi-common-resources/resources/InstallFiles/postgres/$JDBC_JAR $WORKSPACE/rc/lib/$JDBC_JAR
+          fi
+        '''
+      }
+    }
+    stage('Run liquibase') {
+      steps {
+        script {
+          def secretsString = sh(script: '/usr/local/bin/aws ssm get-parameter --name "/aws/reference/secretsmanager/NWCAPTURE-DB-TEST" --query "Parameter.Value" --with-decryption --output text --region "us-west-2"', returnStdout: true).trim()
```

Comment:
This is neat - these all coming from secrets/param store in aws now?  Do any credentials still need to exist in Jenkins?

Reply:
Correct, all from AWS, none from Jenkins